experimental/packages/opentelemetry-instrumentation/README.md

```diff
@@ -263,6 +263,12 @@ If nothing is specified the global registered provider is used. Usually this is
 There might be usecase where someone has the need for more providers within an application. Please note that special care must be takes in such setups
 to avoid leaking information from one provider to the other because there are a lot places where e.g. the global `ContextManager` or `Propagator` is used.
 
+## Instrumentation for ES Modules In NodeJS (experimental)
+
+As the module loading mechanism for ESM is different than CJS, you need to select a custom loader so instrumentation can load hook on the esm module it want to patch. To do so, you must provide the `--experimental-loader=@opentelemetry/instrumentation/hook.mjs --experimental-import-meta-resolve` flag to the `node` binary. Alternatively you can set the `NODE_OPTIONS` environment variable to `NODE_OPTIONS="--experimental-loader=@opentelemetry/instrumentation/hook.mjs --experimental-import-meta-resolve"`.
```

Comment:
is there some limitations/required sequence in case users would like to use other loaders? e.g. a typescript loader,...?

Reply:
It's possible that sequencing matters, but since this is experimental support for now it might be best to get that feedback from users and update guidance from there because we don't know all the variations that might get used.